Unity-Technologies/com.unity.multiplayer.samples.bitesize
# feat: Third Person Character Controller integration with custom IP UI

This PR adds a new Player Prefab for NetworkManager, based off of the Third Person Controller Starter Assets. Previous functionality of movement, collisions, and picking up is intact. The new Input System has been integration for input polling. ClientNetworkAnimator enables the animations of the player to be client-driven.

Included as well is an overhaul on the UI to now include custom IPs, using UI Toolkit.

[MTT-5013](https://jira.unity3d.com/browse/MTT-5013)

Package Manager manifest will target the new Boss Room release containing the new Utilities release, once that is out!

Testing instructions:
* Create a project clone via ParrelSync, or create a build of the game. To test the game via the editor, open up the Bootstrap scene and hit Play.
* Have one instance of the game host, and have other instances join the host.
* Here, validate that all clients see animations being played out from every other client. Test picking up ingredients with E.
